class-components/src/components/DetailSection/DetailSection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { useGetPersonQuery } from '../../store/apiSlice';
+
+const DetailSection: React.FC = () => {
+  const [searchParams] = useSearchParams();
+  const id = searchParams.get('details');
+  const { data, error, isLoading } = useGetPersonQuery(id || '');
+
+  console.log('DetailSection rendered');
+  console.log('Selected item:', data);
+
```

Comment:
Please remove all the `console.log`s from the production version of the application - can be easily checked by Ctrl+F on the PR page